CHANGELOG.md

```diff
@@ -80,6 +80,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Moved `batch_to_device` method from `Accelerator` to `TrainingTypePlugin` ([#10649](https://github.com/PyTorchLightning/pytorch-lightning/pull/10649))
 
 
+- Moved `model_sharded_context` method from `Accelerator` to `TrainingTypePlugin` ([#10886](https://github.com/PyTorchLightning/pytorch-lightning/pull/10886))
```

Comment:
should this be under the `Removed` section, since it's removing `model_sharded_context` from `Accelerator`? `model_sharded_context` already existed on `TrainingTypePlugin`

Reply:
Make sense! will update